egs/madcat_ar/v1/local/prepare_lexicon.py

```diff
@@ -19,7 +19,7 @@
         line_vect = line.strip().split(' ')
         for i in range(1, len(line_vect)):
             characters = list(line_vect[i])
-            characters = " ".join(characters)
+            characters = " ".join([ 'SIL' if char == '|' else char for char in characters])
```

Comment:
please add a comment here to explain what is going on. 
BTW, does the | occur at the begining or end of these things?

Reply:
It occurs at the beginning of the words.